src/backend/distributed/utils/citus_clauses.c

```diff
@@ -162,7 +161,7 @@ ExecuteMasterEvaluableFunctions(Query *query, PlanState *planState)
  * Walks the expression evaluating any node which invokes a function as long as a Var
  * doesn't show up in the parameter list.
  */
-static Node *
+Node *
```

Comment:
I briefly asked him and his response was: `yes, if there are no parameters`
That's why I added the parametrized test.